kolibri/plugins/default_theme/kolibri_plugin.py

```diff
@@ -50,6 +49,7 @@ def theme(self):
             # sign-in page config
             theme_hook.SIGN_IN: {
                 theme_hook.BACKGROUND: static("background.jpg"),
+                theme_hook.DISABLE_SCRIM: False,
```

Comment:
In retrospect, I think a more flexible solution would be to make this option `SCRIM_OPACITY`, and let it take a decimal value between `0` and `1` with a default value of `0.7`.